core-bundle/src/DataContainer/DataContainerBuilderInterface.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Contao\CoreBundle\DataContainer;
+
+use Contao\DataContainer;
+
+interface DataContainerBuilderInterface
```

Comment:
Do we really need this interface? This would prevent us from adding methods in the future.

Reply:
While developing this I first wrote the interface just to have an initial collection of functions that the builder definitely needs. Whether we actually use this (or any of the other) interfaces is open for discussion for me.

Reply:
Yeah I also think that doesn't make a lot of sense. To me, this concept looks a lot like the `PaletteManipulator`.